forge-gui/res/cardsfolder/upcoming/cornered_crook.txt

```diff
@@ -0,0 +1,10 @@
+Name:Cornered Crook
+ManaCost:4 R
+Types:Creature Viashino Warrior
+PT:5/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | OptionalDecider$ You | Execute$ TrigImmediateTrig | TriggerDescription$ When CARDNAME enters the battlefield, you may sacrifice an artifact. When you do, CARDNAME deals 3 damage to any target.
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | Cost$ Sac<1/Artifact> | Execute$ TrigDamage | SpellDescription$ When you do, CARDNAME deals 3 damage to any target.
+SVar:TrigDamage:DB$ DealDamage | NumDmg$ 3 | ValidTgts$ Any
+DeckHints:Type$Artifact
```

Comment:
Probably Needs – this is kind of a dud outside of artifact decks
Probably add the `AI:RemoveDeck:Random` too